webpack/assets/javascripts/foreman_tools.js

```diff
@@ -80,3 +81,22 @@ export function initTypeAheadSelect(input) {
     width: '400px'
   });
 }
+
+// send get forms through turbolinks
+export function tableUpdated(element) {
+  let uri = new URI(window.location.href);
+  let search;
+
+  if (element !== undefined) {
```

Comment:
One of the reasons we can't have multiple implementation of search in the same page is because of the ID selector (see roles and filters). I would like to decouple ourself from that pattern and allow using multiple search fields when appropriate.